### PR TITLE
OpenCV 3.3.0 Update

### LINKED ISSUE #4: Update to OpenCV 3.3

### PR DESCRIPTION
**Pull request by cjlawson02**
Fixes #4 

> Changes for compiling OpenCV 3.3.0 for RoboRio.
> 
> *.sh Changes:
> - OPENCV_VERSION variable changed to 3.3.0
> - GitHub user "Itseez" is old, changed to "opencv"
> 
> Dependency Changes:
> - Numpy wheel is discontinued. Using ipkg instead.
> 
> build.sh Changes:
> - removed wheel unpacking (not needed after numpy ipkg)
> - Commented out JAVA_HOME, not needed for compilation
> - fixed OpenCV-*.tar.gz move command

### CODE REVIEW

File: deps [hunk 1]
```diff
@@ -1,3 +1,3 @@
 http://www.tortall.net/~robotpy/feeds/2017/python36_3.6.0-r1_cortexa9-vfpv3.ipk
 http://www.tortall.net/~robotpy/feeds/2017/python36-dev_3.6.0-r1_cortexa9-vfpv3.ipk
-https://www.tortall.net/~robotpy/wheels/2017/numpy-1.12.0-cp36-cp36m-linux_armv7l.whl
\ No newline at end of file
+https://www.tortall.net/~robotpy/feeds/2017/python36-numpy_1.12.0_cortexa9-vfpv3.ipk
```
Comment (auscompgeek) [inline deps]: Why did you change the permissions on this file?

File: build.sh [hunk 2]
```diff
@@ -38,10 +35,10 @@ function assert_path {
   fi
 }
 
-if [ -z "$JAVA_HOME" ]; then
-  echo "ERROR: JAVA_HOME not set! (maybe you need to do '. /etc/profile', or logout/login)"
-  exit 1
-fi
+# if [ -z "$JAVA_HOME" ]; then
```
Comment (virtuald) [inline build.sh:38]: Hm. We do need a way to make sure that Java is present however (since this repo claims to build java artifacts too, not just python artifacts). What platform are you building on?

**Comment by cjlawson02** [comment]
@auscompgeek Can you explain this more?
@virtuald The repo says to use FIRST’s OpenCV for java (and that it’s broken). Do we need to compile java for 2018?

**Comment by virtuald** [comment]
Ha, so it is. Ok, ignore my comment then.

**Comment by virtuald** [comment]
@Chris2fourlaw what David is referring to is the deps file -- its file permissions have changed in one of your commits, presumably unintentionally (100644 → 100755 ). You must have done a `chmod +x deps` at some point?

**Comment by cjlawson02** [comment]
@virtuald Oh yes. I recursively changed permissions because I was getting a permission error from #6. I’ll commit a patch in a second, as well as fixing a typo in deps.

File: deps [hunk 3]
```diff
@@ -1,3 +1,3 @@
 http://www.tortall.net/~robotpy/feeds/2017/python36_3.6.0-r1_cortexa9-vfpv3.ipk
 http://www.tortall.net/~robotpy/feeds/2017/python36-dev_3.6.0-r1_cortexa9-vfpv3.ipk
-https://www.tortall.net/~robotpy/feeds/2017/python36-numpy_1.12.0_cortexa9-vfpv3.ipk
+http://www.tortall.net/~robotpy/feeds/2017/python36-numpy_1.12.0_cortexa9-vfpv3.ipk
```
Comment (virtuald) [inline deps]: I suppose all of them should be https? Can't think of why not, since it works.

**Comment by virtuald** [comment]
Thanks!
